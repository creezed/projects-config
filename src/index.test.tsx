import { render, screen } from '@testing-library/react';

describe('UserInfo.jsx', () => {
  test('snapshot renders correctly, falsy values', () => {
    render(<div data-testid="button">dfb</div>);
    const element = screen.getByTestId('button');
    expect(element).toMatchSnapshot();
  });
});
