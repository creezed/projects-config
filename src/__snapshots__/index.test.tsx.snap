// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo.jsx snapshot renders correctly, falsy values 1`] = `
<div
  data-testid="button"
>
  dfb
</div>
`;
